rever/activities/conda_forge.xsh

```diff
@@ -143,12 +146,16 @@ class CondaForge(Activity):
 
         # Check if fork exists
         if fork:
+            fork_org = $GITHUB_FORK_ORG or username
             fork_repo = gh.repository(username, feedstock_reponame)
             if fork_repo is None or (hasattr(fork_repo, 'is_null') and
                                      fork_repo.is_null()):
                 print("Fork doesn't exist creating feedstock fork...",
                       file=sys.stderr)
-                repo.create_fork(username)
+                if $GITHUB_FORK_ORG:
+                    repo.create_fork($GITHUB_FORK_ORG)
+                else:
+                    repo.create_fork()
```

Comment:
I'd prefer if the repo.create_fork() call was outside of the if-else.

Reply:
Is that possible? `repo.create_fork` needs to have different call signatures.

Reply:
Oh yeah you are right!